spot_driver/include/spot_driver/api/default_robot_state_client.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#pragma once
+
+#include <bosdyn/client/robot_state/robot_state_client.h>
+#include <spot_driver/api/time_sync_api.hpp>
+#include <spot_driver/interfaces/robot_state_client_interface.hpp>
+
+#include <memory>
+#include <string>
+
+namespace spot_ros2 {
+
+class DefaultRobotStateClient : public RobotStateClientInterface {
```

Comment:
Could we rename this into DefaultRobotStateApi and the super class RobotStateApi? To use the same naming convention of the IK API. Otherwise we could rename the IK API, as long as we keep some consistency.

Reply:
OK